inc/meta-box-html.php

```diff
@@ -14,7 +14,7 @@ public function draw( $field, $form_id = NULL ) {
 			return new WP_Error( 'field_required', 'You need to pass a field array to this method. You passed a '. gettype( $field ) . ' .');
 		}?>
 		<div class="cms-toolkit-wrapper<?php if (isset( $field['class'] )) { echo ' ' . esc_attr( $field['class'] ); } ?>"><?php
-		if ( ! $field['type'] == 'formset' ) {
+		if ( $field['type'] !== 'formset' ) {
```

Comment:
@kurtw For future reference, for some reason, these two lines (the existing line and the one I changed it to) are not equivalent in PHP. Nothing was evaluating as `true` here.